test/distributed/org/apache/cassandra/distributed/impl/Instance.java

```diff
@@ -485,11 +485,10 @@ public void startup(ICluster cluster)
                             GossipHelper.unsafeStatusToNormal(this, (IInstance) peer);
                     });
 
+                    StorageService.instance.setUpDistributedSystemKeyspaces();
```

Comment:
this was a regressions... when not using gossip we didn't register the distributed tables, and we tried to force trace regardless...

right before we gossip tokens around we call this method, so adding it here makes sure the distributed tables work with and without gossip.